recipes/irfinder/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+cd $SRC_DIR/src/irfinder && make && mv irfinder $SRC_DIR/bin
+cd $SRC_DIR/src/trim && make && mv trim $SRC_DIR/bin
+cd $SRC_DIR/src/winflat && make && mv winflat $SRC_DIR/bin
+
+cd $PREFIX
+
+irfinder_home=$PREFIX/opt/irfinder-$PKG_VERSION
```

Comment:
I'm not sure, 

IRFinder is apparently using a perl wrapper script which execute scripts which are in bin/util. That's why I went for the option of putting the whole directory (with bin and utils) in /opt. I could also limit to the bin directory ie:

cp -r $SRC_DIR/bin $irfinder_home

Let me know if I miss a better solution which will not break the dependencies neither clutter the bin directory with them.

IRFinder bin tree:

bin/
├── AdaptorDetect.pl
├── analysisWithLowReplicates.pl
├── analysisWithNoReplicates.pl
├── DESeq2Constructor.R
├── IRFinder
└── util/
    ├── bed-to-intron+exon.pl
    ├── Build-BED-refs.sh
    ├── generateReadsError.pl
    ├── gtf2bed-custom.pl
    ├── IntronExclusion.pl
    ├── irfinder
    ├── IRFinder-BuildRefFromEnsembl
    ├── Mapability
    ├── trim
    ├── warnings
    └── winflat